app/screens/channel/__snapshots__/channel.test.tsx.snap

```diff
@@ -0,0 +1,359 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
How can there be a snapshot file when the channel.test.tsx doesn't have toMatchSnapshot()?

And thank you for not doing toMatchSnapshot().

Reply:
I must have added it and removed it but didn't remove the snapshot file.